colab/static/js/subscriptions.js

```diff
@@ -0,0 +1,32 @@
+$(function (){
+
+  ENTER_BUTTON = 13;
+  searchMaillingLists();
+
+  $('#search_list').on('click', function(event) {
+    searchMaillingLists();
+  });
+
+  $('#list_name').on('keypress', function(event) {
```

Comment:
@matheus-fga I don't know if it was only on my environment, but it this feature had a tiny little bug: when I typed 'lista', it was searching for 'list' only (missing the last character). I discovered this weekend that jQuery has this 'input' event, that makes it easier to handle 'keypress'-like events: https://github.com/colab/colab/blob/detach_super_archives/colab/static/js/subscriptions.js#L7. It should be fine now...